cli.py

```diff
@@ -123,19 +123,29 @@
                 headers = {"Authorization": f"token {github_access_token}"}
                 response = requests.get("https://api.github.com/user", headers=headers)
                 if response.status_code == 200:
-                    scopes = response.headers.get("X-OAuth-Scopes")
+                    o_auth_scopes_header = "X-OAuth-Scopes"
+                    scopes = response.headers.get(o_auth_scopes_header, [])
                     if "public_repo" in scopes or "repo" in scopes:
                         click.echo(
                             click.style(
                                 "✅ GitHub access token has the required permissions.",
                                 fg="green",
                             )
                         )
+                    elif o_auth_scopes_header not in response.headers:
+                        install_error = True
+                        click.echo(
+                            click.style(
+                                "❌ Response is lack of X-OAuth-Scopes, please make sure you use OAuth token.",
+                                fg="red",
+                            )
+                        )
```

Comment:
**Suggestion:** The variable `install_error` is set to `True` when the required scopes are not present or the `X-OAuth-Scopes` header is missing, but there is no further action taken based on this variable within the provided code context. If `install_error` is intended to control flow or trigger additional actions, ensure that it is used effectively or consider removing it if unnecessary. [possible issue]
```suggestion
                    elif o_auth_scopes_header not in response.headers:
                        click.echo(
                            click.style(
                                "❌ No 'X-OAuth-Scopes' found in the response. Please ensure you are using an OAuth token.",
                                fg="red",
                            )
                        )
                        return False  # Assuming returning False indicates an error
```

Reply:
I would keep in this code because it's exist way to handle unexpected scenario